src/main/java/core/basesyntax/Main.java

```diff
@@ -0,0 +1,12 @@
+package core.basesyntax;
+
+import java.util.Arrays;
+
+public class Main {
```

Comment:
if tests are present, no need to push Main next time. Of course, if it's not in the task description.